trade-finance-manager-api/src/v1/rest-mappings/mappings/facilities/mapFacilityStage.test.js

```diff
@@ -0,0 +1,62 @@
+const { TFM_FACILITY_STAGE } = require('@ukef/dtfs2-common');
+const { mapBssEwcsFacilityStage, mapGefFacilityStage } = require('./mapFacilityStage');
+
+describe('mapGefFacilityStage', () => {
+  describe('when a tfm facility stage is passed in', () => {
+    it('should return the tfm facility stage', () => {
+      const result = mapGefFacilityStage(true, TFM_FACILITY_STAGE.RISK_EXPIRED);
+      expect(result).toEqual(TFM_FACILITY_STAGE.RISK_EXPIRED);
+    });
+  });
+
+  describe('when facilityStage is true boolean', () => {
+    it('should return facilityStage as `Issued`', () => {
+      const result = mapGefFacilityStage(true);
+      expect(result).toEqual('Issued');
+    });
+  });
+
+  describe('when facilityStage is false boolean', () => {
+    it('should return facilityStage as `Commitment`', () => {
+      const result = mapGefFacilityStage(false);
+      expect(result).toEqual('Commitment');
+    });
+  });
+});
+
+describe('mapBssEwcsFacilityStage', () => {
+  describe('when a tfm facility stage is passed in', () => {
+    it('should return the tfm facility stage', () => {
+      const result = mapBssEwcsFacilityStage(true, TFM_FACILITY_STAGE.RISK_EXPIRED);
+      expect(result).toEqual(TFM_FACILITY_STAGE.RISK_EXPIRED);
+    });
+  });
+
+  describe('when facilityStage is `Issued`', () => {
+    it('should return facilityStage as `Issued`', () => {
+      const result = mapBssEwcsFacilityStage('Issued');
+      expect(result).toEqual('Issued');
+    });
+  });
+
+  describe('when facilityStage is `Unconditional`', () => {
+    it('should return facilityStage as `Issued`', () => {
+      const result = mapBssEwcsFacilityStage('Unconditional');
+      expect(result).toEqual('Issued');
+    });
+  });
+
+  describe('when facilityStage is `Unissued`', () => {
+    it('should return facilityStage as `Commitment`', () => {
+      const result = mapBssEwcsFacilityStage('Unissued');
+      expect(result).toEqual('Commitment');
+    });
+  });
+
+  describe('when facilityStage is `Conditional`', () => {
+    it('should return facilityStage as `Commitment`', () => {
+      const result = mapBssEwcsFacilityStage('Conditional');
+      expect(result).toEqual('Commitment');
+    });
+  });
+});
```

Comment:
Can we use constants for all issued/stage types please

Reply:
Yep good spot, have changed

Reply:
Can also destructure like we do in the main function 😎

Reply:
@ttbarnes I've changed the test descriptions to do that. I'm destructuring the Portal constants at the top of the file already:  
`const { ISSUED, UNCONDITIONAL, UNISSUED, CONDITIONAL } = CONSTANTS.FACILITIES.FACILITY_STAGE_PORTAL`

But I can't also destructure the Tfm constants as there is some overlap, e.g. also doing: 
`const { ISSUED, COMMITMENT, RISK_EXPIRED } = TFM_FACILITY_STAGE`

Would lead to a constant clash on ISSUED. Due to the importance of distinguishing the portal vs tfm contsants here I think keeping the tfm constants as e.g. `TFM_FACILITY_STAGE.ISSUED` is also helpful.

Reply:
Makes sense, thanks @BethThomas141